assets/css/edd-admin.css

```diff
@@ -2431,6 +2414,57 @@ body.download_page_edd-reports {
 	margin-bottom: 20px;
 }
 
+div.edd-reports-export {
+	display: grid;
+	display: -ms-grid;
+	grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+	grid-gap: 20px;
+}
+
+.edd-reports-wrapper .postbox {
+	margin-bottom: 0;
+}
+
+.edd-export-sales-earnings {
+	grid-template-columns: 1fr;
+}
+
+[class^="edd-export"] .edd-from-to-wrapper {
+	display: flex;
+	margin-right: 0;
+	margin-bottom: 16px;
+	width: 100%;
+}
+
+.download_page_edd-reports .inside input[type="text"] {
```

Comment:
You could replace this with `.edd-reports-export .edd-from-to-wrapper input`, which might feel better. Then add the `flex-grow` to the input parent `span` elements:

```css
.edd-reports-export .edd-from-to-wrapper span {
    flex-grow: 1;
}
```
I think both the width on the inputs and the flex-grow are needed to get that whole thing full width.

Reply:
(Instead of `span` you could target the IDs of the `span` elements....I was hoping there was a way to add a class to those but it does not appear to be possible.)